.github/CODEOWNERS

```diff
@@ -2173,4 +2173,3 @@ spec/uploaders/simple_forms_api/ @department-of-veterans-affairs/platform-va-pro
 spec/uploaders/supporting_evidence_attachment_uploader_spec.rb @department-of-veterans-affairs/disability-benefits @department-of-veterans-affairs/backend-review-group
 spec/uploaders/uploader_virus_scan_spec.rb @department-of-veterans-affairs/backend-review-group
 spec/uploaders/validate_pdf_spec.rb @department-of-veterans-affairs/disability-benefits @department-of-veterans-affairs/backend-review-group
-# Place your entry above this comment in alphabetical order.
```

Comment:
Would you mind re-adding this comment? (🫣 srry, I know it messes with the `sort -u`, but it does help)

Reply:
ah shoot yea I just did `sort -u` and `git add` didnt even read/look at it. We should add a git commit hook that sorts it on commit (ignoring the comments), Im sure there is a way.... problem for another day though, I will manually move